c2corg_ui/static/js/documentediting.js

```diff
@@ -50,16 +50,16 @@ app.module.directive('appDocumentEditing', app.documentEditingDirective);
  * @param {app.Authentication} appAuthentication
  * @param {ngeo.Location} ngeoLocation ngeo Location service.
  * @param {app.Alerts} appAlerts
+ * @param {angularGettext.Catalog} gettextCatalog Gettext catalog.
```

Comment:
The param comment should be placed in the same order then in the function arguments list.